drivenRangeChecker.cpp

```diff
@@ -0,0 +1,49 @@
+#include "currentRanges.hpp"
+#include <algorithm> 
+#include <iostream>
+
+bool checkForContinuousRange(int currentIndex, int nextIndex)
+{
+   if(((nextIndex - currentIndex) == 1) || ((nextIndex - currentIndex) == 0))
+   {
+	   return true;
+   }
+   return false;
+}
+
+void checkForRangeContinuity(std::vector<int> sortedRange, int currentIndex, currentRanges& currentRange)

```

Comment:
The function name sounds like 'check for ...'
Not only does it check, it also fills the currentRange. Try a name that indicates its purpose better.
Difficulty in naming? Maybe It needs to be split into two functions?

Reply:
Renamed accrodingly.